### ISSUE TITLE
Type errror in _encode_hybi when using the --heartbeat parameter

### ISSUE THREAD

**Issue by Eijebong** [novnc/websockify#333]
When using the `--heartbeat` option, every N seconds, websockify gives us a nice `TypeError: can't concat bytes to str` because the value sent from `send_ping()` is a `str` and not a `bytes`.

```
  File "/opt/websockify/websockify/websocketproxy.py", line 180, in do_proxy
    self.send_ping()
  File "/opt/websockify/websockify/websockifyserver.py", line 211, in send_ping
    self.request.ping(data)
  File "/opt/websockify/websockify/websocket.py", line 447, in ping
    self._sendmsg(0x9, data)
  File "/opt/websockify/websockify/websocket.py", line 688, in _sendmsg
    frame = self._encode_hybi(opcode, msg)
  File "/opt/websockify/websockify/websocket.py", line 767, in _encode_hybi
    return header + buf
TypeError: can't concat bytes to str
```

Here's an example of fix. It might need to be applied to pong too.

```diff
diff --git a/websockify/websocketproxy.py b/websockify/websocketproxy.py
index 6c17c10..febf7b3 100644
--- a/websockify/websocketproxy.py
+++ b/websockify/websocketproxy.py
@@ -177,7 +177,7 @@ Traffic Legend:
                 now = time.time()
                 if now > self.heartbeat:
                     self.heartbeat = now + self.server.heartbeat
-                    self.send_ping()
+                    self.send_ping("".encode())
 
             if tqueue: wlist.append(target)
             if cqueue or c_pend: wlist.append(self.request)

```

**Comment by CendioOssman** [comment]
I'll have a look. Note that this affects Python 3, but not Python 2.

**Comment by CendioOssman** [comment]
Fixed in 3b2a0a5884730291a603538d1b11fc308c2028a3.

**Comment by Eijebong** [comment]
Thanks :)
